HomeAssistant.xcodeproj/project.pbxproj

```diff
@@ -6522,6 +6522,7 @@
 				CODE_SIGN_ENTITLEMENTS = "Configuration/Entitlements/Extension-ios.entitlements";
 				"CODE_SIGN_ENTITLEMENTS[sdk=macosx*]" = "Configuration/Entitlements/Extension-catalyst.entitlements";
 				INFOPLIST_FILE = Sources/Extensions/Share/Resources/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
```

Comment:
this should be set in the xcconfig, rather than the xcodeproj

(when in doubt, do it in the xcconfig - makes reviewing/history a lot easier, and reduces a lot of churn in the proj file)

Reply:
Yeah, some were overridden to be newer, but all less than iOS 15. We should be able to delete all xcodeproj-level deployment targets.